### ISSUE TITLE
AuthMe Plugin issue (Not having the permission to perform commands)

### ISSUE THREAD

**Issue by Bass-Buster** [Xephi/AuthMeReloaded#919]
### What behaviour is observed:
Regular players, the ones that have no OP or the ones taht don't have the "*" permission cannot use authme commands due to "You don't have permission to use this command!". Therefore no possibility to register or login.

### What behaviour is expected:
I expected the plugin to work normally as it usually did.

### Steps/models to reproduce:
All AuthMe commands.

### Plugin list:
PlaceHolderAPI
PlugMan
SkinsRestorer
ViaVersion
Tablist
CoreProtect
WorldEdit
SimpleTrading
NBTEditor
ClearLag
BuycraftX
Votifier
Multiverse-Core
ProtocolLib
UberEnchant
ObsidianBreaker
PermissionsEx
McMMO
Vault
ServerListPlus
GAListener
MassiveCore
Rankup
CreativeGates
HolographicDisplays
WorldGuard
PlayerVaults
MassiveBooks
KeepDeathDrop
MyCommand
MassiveHat
Essentials
CompatNoCheatPlus
Jobs
Auctions
CrateReloaded
NoCheatPlus
KitAdder
ChopTree3
EssentialsChat
JLib

### Environment description
Spigot 1.9.4 server with MySQL database support. Hosted.

### AuthMe build number:
AuthMeReloaded v5.2-BETA3

### Error Log:
There are no errors being displayed.

### Configuration:
It's the default Config, because i deleted the folder completely ant the plugin generated a fresh folder with the configuration inside it.

### NOTE:
Same plugins of latest versions were present before the issue. Nothing was changed among the plugins as far as I know. No new ones installed, no old ones deleted. The issue just kind of came out of nowhere.

**Comment by ljacqu** [comment]
Hello, if you've updated from an older AuthMe version it's possible that you need to adjust your permission nodes. We used to have things like `authme.login` and now they're along the format of `authme.player.login`. An up-to-date list of permission nodes can be found at https://github.com/AuthMe/AuthMeReloaded/blob/master/docs/permission_nodes.md

**Comment by Bass-Buster** [comment]
I had the same version of AuthMe installed on the same server with the same plugins and it worked fine. And then it suddenly didn't work. I tried making the group in permissions file - no succes, I tried reinstalling the whole plugin (although it's the same latest version) - no success.
No plugins were installed, deleted or updated between the two periods of "AuthMe works fine" and "AuthMe tells players they don't hav epermission".
I used the up-to-date permissions.
